docs/sources/developers/plugins/migration-guide.md

```diff
@@ -0,0 +1,151 @@
+
+Since Grafana 7 introduced [signing of backend plugins](https://grafana.com/docs/grafana/latest/plugins/plugin-signature-verification/), community plugins won’t load by default if they’re unsigned.
+
+To learn more, refer to [Backend plugins](https://grafana.com/docs/grafana/latest/developers/plugins/backend/).
+
+## Why should I migrate my plugin?
+
+There are several benefits in using the new plugin platform.
+
+- **Better performance:** Components written in React are more responsive.
+- **Field options**: By migrating to the new data frame format, you can leverage the new field options to let users fine-grained customization of the data and display options.
+
+## Compatibility between Grafana versions
+
+A plugin developed for Grafana 6 will work for Grafana 7. However, plugins developed using the new plugin platform in Grafana 7 will only work for Grafana 7 and up.
+
+### Interoperability between data formats
+
+Grafana detects the data format sent by the data source and transforms it for the panel, if needed.
+
+For example:
+
+- A legacy panel with data source that returns data frames: Grafana converts the response to the legacy format.
+- A legacy data source with a panel using data frames: Grafana converts the response to the data frame format.
+- If both panel and data source uses the same format, no transformations are made. Data is passed as is.
+
+### target and jsonData are unchanged
+
+The query model, `target`, and the configuration model, jsonData,  are still the same. This means that if you use the same query model and configuration for your plugin, the migrated plugin will use existing queries and configuration. You don’t have to worry about breaking existing dashboards.
+
+## How do I migrate from an Angular plugin to a React plugin?
+
+If you’re looking to migrate a plugin to the new plugin platform, we recommend that you release it under a new major version, and that you keep a release branch for the previous version to be able to roll out patch releases for versions prior to Grafana 7.
+
+While there’s no 1-to-1 migration path from a Angular plugin to the new React platform, from early adopters, we’ve learned that one of the easiest ways to migrate is to create a new plugin from one of the templates provided by Grafana Toolkit, and then move the existing code into the new plugin.
+
+An example of how you could do this is to create a new branch called `migrate-to-react`, delete everything and start with one of the plugin templates.
+
+### Migrate a panel plugin
+
+In <7.0, you would export a MetricsPanelCtrl from module.ts.
+
+```ts
+import { MetricsPanelCtrl } from 'grafana/app/plugins/sdk';
+
+class MyPanelCtrl extends MetricsPanelCtrl {
+  // ...
+}
+
+export { MyPanelCtrl as PanelCtrl };
+```
+
+In >=7.0, plugins now export a PanelPlugin from module.ts
```

Comment:
The less than or equal to symbols in prose are very hard to parse. Please do not use them. Use words like "before" instead.